Jellyfin.Data/Enums/HomeSectionType.cs

```diff
@@ -53,6 +53,11 @@ public enum HomeSectionType
         /// <summary>
         /// Continue Reading.
         /// </summary>
-        ResumeBook = 9
+        ResumeBook = 9,
+
+        /// <summary>
+        /// Latest Premier.
+        /// </summary>
+        LatestPremier = 10
```

Comment:
Maybe "NewReleases"? `LatestMedia` was a really terrible name considering how ambiguous "latest" is...